main.tf

```diff
@@ -6,9 +6,26 @@ data "google_project" "my_host_project" {
   project_id = var.host_project_id
 }
 
+data "google_folders" "my-org-folders" {
+  parent_id = "organizations/${var.organization_id}"
+}
+
+data "google_folders" "app_script_folders" {
+  count = length(local.system_gsuite_folders) == 0 ? 0 : 1
+  parent_id = local.system_gsuite_folders[0]
+}
+
 locals {
+  # Get gsuite folder in organization 
+  system_gsuite_folders = [for each in data.google_folders.my-org-folders.folders : each.name if each.display_name == "uptycs"]
```

Comment:
"uptycs"?
Did you check-in your test code? :-)

Reply:
yes